doc/stdlib/file.rst

```diff
@@ -81,6 +81,13 @@
 
    Change the permissions mode of ``path`` to ``mode``\ . Only integer ``mode``\ s (e.g. 0o777) are currently supported.
 
+.. function:: chown(path, owner, group =-1)
```

Comment:
I think the genstdlib script might be sensitive to this matching the signature in the docstring exactly, including spacing